repos/system_upgrade/el9toel10/actors/sssd/sssdchecks/libraries/sssdchecks.py

```diff
@@ -0,0 +1,33 @@
+from leapp import reporting
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def check_config(model):
+    if not model:
+        return
+
+    # If sss_ssh_knownhostsproxy was not configured, there is nothing to do
+    if not model.ssh_config_files:
+        return
+
+    summary = (
+        'SSSD\'s sss_ssh_knownhostsproxy tool is replaced by the more '
+        'reliable sss_ssh_knownhosts tool. SSH\'s configuration will be updated '
+        'to reflect this by updating every mention of sss_ssh_knownhostsproxy by '
+        'the corresponding mention of sss_ssh_knownhosts, even those commented out.\n'
+        'SSSD\'s ssh service will be enabled if not already done.\n'
+        'The following files will be updated:{}{}'.format(
```

Comment:
text formatting nitpick - just in case we do any other change, possibly good to modify too, but no required.
The reason is that the report is presented by various UIs (cockpit, satellite, ..., and txt file). In various webUIs the message looks weird with various short lines. In case of the txt file, NL is not so visible and to improve readability it's better to add `\n\n` double NLs to have a separate paragraph.

```suggestion
        'the corresponding mention of sss_ssh_knownhosts, even those commented out. '
        'SSSD\'s ssh service will be enabled if not already done.\n\n'
        'The following files will be updated:{}{}'.format(
```